node/src/main/scala/com/wavesplatform/database/LevelDBWriter.scala

```diff
@@ -626,7 +629,36 @@ class LevelDBWriter(writableDB: DB, spendableBalanceChanged: Observer[(Address,
     db.get(Keys.transactionHNById(txId)).map(_._1)
   }
 
-  override def addressTransactions(address: Address, types: Set[TransactionParser], fromId: Option[ByteStr]): CloseableIterator[(Height, Transaction)] = readStream { db =>
+  override def portfolioNFT(address: Address, from: Option[IssuedAsset]): CloseableIterator[IssueTransaction] = readStream { db =>
+    val assetIdStream: CloseableIterator[IssuedAsset] = db
+      .get(Keys.addressId(address))
+      .fold(CloseableIterator.empty[IssuedAsset]) { id =>
+        val addressId = AddressId @@ id
+
+        val iter = db.get(Keys.assetList(addressId)).iterator
```

Comment:
This will return asset ids from oldest to newest, but we need to have them in the reverse order.